server/routes/appointments/create-and-edit/appointmentJourney.ts

```diff
@@ -1,12 +1,18 @@
 import { YesNo } from '../../../@types/activities'
-import { AppointmentRepeatPeriod } from '../../../@types/activitiesAPI/types'
+import { AppointmentRepeatPeriod } from '../../../@types/appointments'
 
 export enum AppointmentType {
   INDIVIDUAL = 'INDIVIDUAL',
   GROUP = 'GROUP',
 }
 
-export type CreateAppointmentJourney = {
+export enum AppointmentJourneyMode {
+  CREATE = 'CREATE',
+  EDIT = 'EDIT',
+}
+
+export type AppointmentJourney = {
+  mode?: AppointmentJourneyMode
```

Comment:
Does this have to be nullable? It's always specified on each of the start handlers I think